### DISCUSSION TITLE
How to install dashboards in my own Grafana 7 docker

### DISCUSSION THREAD

**Discussion by tgerighty** [teslamate-org/teslamate#1755] [Getting started] [answered]
Hi. - I followed the instructions and imported all the dashboards into my own Grafana docker instance but I get an error on the grafana-map-panel not being installed.

Problem is that I can't find this panel, I can however find grafana-worldmap-panel but that doesn't appear to be the right one. Can you advise which plugins I should add to use the dashboards in my own grafana.

Error:
grafana-cli plugins install grafana-map-panel
Error: ✗ Failed to find requested plugin, check if the plugin_id is correct: 404 not found error

![image](https://user-images.githubusercontent.com/30463087/123399100-22c6cb00-d59c-11eb-94d2-0febfcb400e0.png)

**Comment by adriankumpf** [comment, marked as answer]
Check out the instruction from the "Manuall installation" guide: https://docs.teslamate.org/docs/installation/debian#requirements
The Dockerfile TeslaMate uses might also be helpful: https://github.com/adriankumpf/teslamate/blob/master/grafana/Dockerfile

**Comment by tgerighty** [comment]
The manual install doesn't really specify the plug-ins but thanks for the Dockerfile which lists them all! 

`grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install pr0ps-trackmap-panel 2.1.2 && \
    grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install natel-discrete-panel 0.1.0 && \
    grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" plugins install grafana-piechart-panel 1.6.1 && \
    grafana-cli --pluginsDir "${GF_PATHS_PLUGINS}" --pluginUrl https://github.com/panodata/grafana-map-panel/releases/download/0.15.0/grafana-map-panel-0.15.0.zip plugins install grafana-worldmap-panel-ng
`
